amundsen_application/static/js/components/common/Badges/styles.scss

```diff
@@ -1,10 +1,25 @@
 // Copyright Contributors to the Amundsen project.
 // SPDX-License-Identifier: Apache-2.0
-
 @import 'variables';
-
 .clickable-badge {
   &:hover {
     cursor: pointer;
   }
-}
+  .label {
+    overflow: hidden;
+    padding: 0 0;
+  }
+  .badge-overlay-primary {
+    font-size: 14px;
+    text-align: center;
+    display: inline-block;
+    padding: .2em .6em .3em;
```

Comment:
We normally use pixel sizes. They are not great for accessibility, but still what we have right now.

If you could try to make them relate to the $spacer-size, as $spacer-1, $spacer-2 and so on, much better!

Reply:
The em units were used on .label so I was trying to keep that consistency

Reply:
Feel free to leave then as ems then!